src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -484,6 +484,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         // Tell the DX Engine to notify us when the swap chain changes.
         dxEngine->SetCallback(std::bind(&TermControl::SwapChainChanged, this));
 
+        dxEngine->SetRetroTerminalEffects(_settings.RetroTerminalEffect());

```

Comment:
We probably also want to do this in `UpdateSettings`, so that changing the setting will hot-reload the setting

Reply:
Can we make a task for this? We would need to do setup outside of normal dxEngine setup path to support hot reloading.

Reply:
Oh, that's a good point. I can make that follow-up